README.md

```diff
@@ -1,6 +1,15 @@
 UI Application for camptocamp.org v6
 ====================================
 
+Requirements
+------------
+
+ * Python 2.7
+ * Node (4.0 or higher)
```

Comment:
Why do we need a node version so new?
I have been using v0.10.25 (from Ubuntu 15.10) without any issue.